### PR TITLE
Enable uploads to S3 bucket using s3cmd

### PR DESCRIPTION
**Pull request by NyakudyaA**
* Enable upload to s3 bucket
* Add env `DUMP_ARGS` that allows users to specify dump options. Useful for people who need SQL dumps etc
* Add env variable `DB_TABLES` which allows dumping individual tables of a db

ping @lucernae can you take a look at this one. They are a couple of todo in there to enhance it but it should work as is

### CODE REVIEW

File: scripts/start.sh [hunk 1]
```diff
@@ -57,6 +57,15 @@ if [[ -f "${PG_ENV}" ]]; then
 fi
 
 echo "
+export HOST_BASE=$HOST_BASE
+export HOST_BUCKET=$HOST_BUCKET
+export DEFAULT_REGION=$DEFAULT_REGION
+export SSL_SECURE=$SSL_SECURE
+export ACCESS_KEY_ID=$ACCESS_KEY_ID
+export SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY
+export EXTRA_CONF=$EXTRA_CONF
```
Comment (lucernae) [inline scripts/start.sh:66]: Can EXTRA_CONF be multiline?
Not sure how it can fit inside this statement without heredoc.

**Review by lucernae — COMMENTED** [review]
We should find a way how to test this in travis/github action with given s3/minio credentials

**Comment by lucernae** [comment]
About the testing, I made unittest for the s3 push in this branch:
https://github.com/kartoza/docker-pg-backup/tree/s3cmd

You can pull it to include it in this branch.

**Comment by NyakudyaA** [comment]
> About the testing, I made unittest for the s3 push in this branch:
> https://github.com/kartoza/docker-pg-backup/tree/s3cmd
> 
> You can pull it to include it in this branch.

Thanks, I will do so

**Pull request merged**